src/main/java/com/internship/frejaeidjmeterplugin/jmeter/visualizer/gui/GraphPanel.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.internship.frejaeidjmeterplugin.jmeter.visualizer.gui;
+
+import java.awt.Color;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import org.knowm.xchart.QuickChart;
+import org.knowm.xchart.XChartPanel;
+import org.knowm.xchart.XYChart;
+
+/**
+ *
+ * @author User
+ */
+public class GraphPanel {
+
+    private XYChart chart;
+    private XChartPanel chartPanel;
+    List<Double> successCount;
+    List<Double> failCount;
+    List<Double> count;
+
+    public GraphPanel(String title, String xAxisName, String yAxisName) {
+        double[] initData = new double[1];
+        initData[0] = 0;
+        chart = QuickChart.getChart(title, xAxisName, yAxisName,"Failed", initData, initData);
+        chart.addSeries("Delivered", initData, initData);
```

Comment:
Move this chart operations in separate private method describing what is done here.